Sources/JetStream/JetStreamContext.swift

```diff
@@ -67,6 +69,12 @@ extension JetStreamContext {
 
         return try decoder.decode(Response<T>.self, from: payload)
     }
+
+    internal func request(_ subject: String, message: Data? = nil) async throws -> NatsMessage {
```

Comment:
Keep in mind we also have this:

```
    public func request<T: Codable>(
        _ subject: String, message: Data? = nil
    ) async throws -> Response<T> {
        let data = message ?? Data()
        let response = try await self.client.request(
            data, subject: "\(self.prefix).\(subject)", timeout: self.timeout)

        let decoder = JSONDecoder()
        guard let payload = response.payload else {
            throw JetStreamRequestError("empty response payload")
        }

        return try decoder.decode(Response<T>.self, from: payload)
    }
    ```

Reply:
I do, that's added because I wanted to have a proxy with setting api subject for fetching standard `NatsMessage` without any JSON decoding - this is useful for implementing direct get , which does not return a JSON